chain/client/src/stateless_validation/chunk_validator/mod.rs

```diff
@@ -428,6 +435,7 @@ fn validate_receipt_proof(
     Ok(())
 }
 
+#[allow(clippy::large_enum_variant)]
```

Comment:
Huh? Did we reach a threshold with the new fields in chunk extra, or how did this trigger suddenly?